### ISSUE TITLE
block comment: insert leading '* ' 

### ISSUE THREAD

**Issue by Tazi0** [zed-industries/zed#15507]
### Check for existing issues

- [X] Completed

### Describe the feature

There currently is no "formatting" for code blocks when pressing enter. There is a completion which is nice, but it would like to see some auto addition when clicking enter like vscode.

https://github.com/user-attachments/assets/74a4e896-992d-4565-b297-12e2f171f032

**Comment by notpeter** [comment]
To be clear, are you looking for Zed to auto add a leading ` *` when pressing enter in a multi-line comment or something more?

I thought there was another issue for Zed to auto-generate doc-strings with params/types from a template but I couldn't find it.

**Comment by Tazi0** [comment]
@notpeter yep that's correct, i could quite phrase it correctly but that's the goal of this issue

**Comment by notpeter** [comment]
This has been fixed! Enjoy!
